### ISSUE TITLE
About image enhancement code and UVOT400 training configuration file

### ISSUE THREAD

**Issue by aqizhoua** [BasitAlawode/UVOT400#9]
In which folder is the image enhancement code? Is there a configuration file in the code about how to use UVOT400 for training? For example, I use DropTrack as the tracker, use the training set of UVOT400 for training, and use the test set of UVOT400 for reasoning.Thanks.

**Comment by BasitAlawode** [comment]
The image enhancement code will be released as soon as possible. 

We already provided the train:test split of the dataset. As such, there is no special configuration file to use UVOT400 for training trackers.
